source/develop/developer-guide/vdsm/log-files.html.md

```diff
@@ -36,6 +36,15 @@ Since oVirt 3.5.0 (VDSM 4.16.0), VDSM does not enable the overly verbose libvirt
 
 ## Reading the log
 
+### Verbosity and debug information
+
+Vdsm produces a huge amount of logging, and to improve the readability and to ease the troubleshooting, effort are ongoing to make the output tunable, to show messages from only some selected subsystems. As part as this effort, Vdsm gained the ability to change the log levels at runtime, without requiring a restart of the daemon. For example, to enable all the debug logs for the `virt` subsystem:
+
```

Comment:
also please fill the pull request template, it's missing signature accepting contribution guidelines